cd/mxnet_lib/mxnet_lib_pipeline.groovy

```diff
@@ -75,9 +75,9 @@ def get_stash(mxnet_variant) {
 // The environment corresponds to the docker files in the 'docker' directory
 def get_environment(mxnet_variant) {
   if (mxnet_variant.startsWith("cu")) {
-    return "publish.centos7_gpu_${mxnet_variant}"
+    return "centos7_gpu_${mxnet_variant}"
```

Comment:
the `_gpu` infix doesn't seem necessary either

Reply:
It's for consistency with the existing naming scheme. The name used here must match the name in the docker-compose.yml